source/platforms/github/_tests/_github_api.test.ts

```diff
@@ -113,13 +113,24 @@ new file mode 0
 
   it("getDangerCommentIDs ignores comments not marked as generated", async () => {
     api.getAllOfResource = await requestWithFixturedJSON("github_inline_comments_with_danger.json")
-    api.getUserID = () => new Promise<number>((r) => r(20229914))
+    api.fetch = jest.fn().mockReturnValue({ ok: true })
```

Comment:
This was failing because it was making an API call (i.e., not mocked) which isn't allowed so I just included it in this PR 😅